Tasks/KubernetesV1/task.json

```diff
@@ -123,6 +123,14 @@
             "groupName": "kubernetesCluster",
             "helpMarkDown": "Set the namespace for the kubectl command by using the –namespace flag. If the namespace is not provided, the commands will run in the default namespace."
         },
+        {
+            "name": "kubernetesAccessProfileType",
+            "type": "boolean",
+            "label": "Get cluster admin credentials",
+            "defaultValue": "false",
+            "groupName": "kubernetesCluster",
+            "helpMarkDown": "Get cluster administrator credentials instead of default cluster user credentials."
```

Comment:
Add visible rule. this logic applies only for AKS
`"visibleRule": "connectionType = Azure Resource Manager"`